examples/api/unocss.config.js

```diff
@@ -70,7 +70,7 @@ export default defineConfig({
     }
   ],
   shortcuts: {
-    btn: `select-none outline-none shadow-md p-2 rd-1 text-primaryText border-none font-400 dark:font-600
+    btn: `select-none outline-none cursor-pointer shadow-md p-2 rd-1 text-primaryText border-none font-400 dark:font-600
```

Comment:
btn by default shouldn't have `cursor: pointer`, native apps don't usually set it
```suggestion
    btn: `select-none outline-none shadow-md p-2 rd-1 text-primaryText border-none font-400 dark:font-600
```

Reply:
Just want to say that looking at most web based apps and also websites I use, like Chrome, Edge, Steam, Discord, vscode, Google, GitHub, ... are using cursor pointer on buttons by default

I prefer it a bit more but yeah, most native apps don't do this

Reply:
yeah it is kinda a preference thing, I usually prefer to stick to native behavior when possible.